src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandlerTest.java

```diff
@@ -116,7 +116,8 @@ public void it_can_handle_callback() {
                         Event.REQUEST_RESPONDENT_REVIEW,
                         Event.LIST_CASE,
                         Event.EDIT_CASE_LISTING,
-                        Event.ADJOURN_HEARING_WITHOUT_DATE
+                        Event.ADJOURN_HEARING_WITHOUT_DATE,
+                        Event.END_APPEAL
```

Comment:
could you please update the other test case to verify triggering the home-office-api.

Reply:
"END_APPEAL" added to other test